vector/src/main/java/im/vector/app/features/matrixto/SpaceCardRenderer.kt

```diff
@@ -55,7 +55,7 @@ class SpaceCardRenderer @Inject constructor(
             if (spaceSummary.isPublic) {
                 inCard.matrixToAccessText.setTextOrHide(stringProvider.getString(R.string.public_space))
                 inCard.matrixToAccessImage.isVisible = true
-                inCard.matrixToAccessImage.setImageResource(R.drawable.ic_public_room)
+                inCard.matrixToAccessImage.setImageResource(R.drawable.ic_tchap_public_room)
```

Comment:
@Claire1817 can you please keep the Element name for this icon, we should just replace the xml content of this icon.